VPC-Section-Formatting (#141)
C2Devel/terraform-provider-croccloud: docs/resources/vpc.md

## Example Usage

> ### Basic #xample

Reviewer: ```suggestion
### Basic Example
```

}
```

## Argument reference

The following arguments are supported:

* `cidr_block` - (Optional, Forces new resource, String) The IPv4 CIDR block for the VPC.
* `enable_dns_support` - (Optional, Boolean) A flag to enable or disable the DNS support in the VPC.
    * _Default value:_ `true`
* `tags` - (Optional, Map of strings) Key-value pairs to assign to the resource. If a provider [default_tags configuration block][default-tags] is used, tags with matching keys will overwrite those defined at the provider level.

## Attribute reference

### Supported attributes

In addition to all arguments above, the following attributes are exported:

* `arn` - (String) The Amazon Resource Name (ARN) of the VPC.

> * `default_network_acl_id` - (String) The ID of the network ACL created by default on VPC creation.

Reviewer 2: Тут еще потерялся атрибут `dhcp_options_id`, добавь пожалуйста. В `aws_default_vpc` тоже

* `default_security_group_id` - (String) The ID of the security group created by default on VPC creation.

> * `default_route_table_id` - (String) The ID of the route table created by default on VPC creation.

Reviewer 2: Нужно поменять местами

}
```

## Argument reference

The following arguments are supported:

* `cidr_block` - (Optional, Forces new resource, String) The IPv4 CIDR block for the VPC.
* `enable_dns_support` - (Optional, Boolean) A flag to enable or disable the DNS support in the VPC.
    * _Default value:_ `true`
* `tags` - (Optional, Map of strings) Key-value pairs to assign to the resource. If a provider [default_tags configuration block][default-tags] is used, tags with matching keys will overwrite those defined at the provider level.

## Attribute reference

### Supported attributes

In addition to all arguments above, the following attributes are exported:

* `arn` - (String) The Amazon Resource Name (ARN) of the VPC.
* `default_network_acl_id` - (String) The ID of the network ACL created by default on VPC creation.
* `default_security_group_id` - (String) The ID of the security group created by default on VPC creation.
* `default_route_table_id` - (String) The ID of the route table created by default on VPC creation.
* `id` - (String) The ID of the VPC.

> * `instance_tenancy` - (String) The allowed tenancy of instances.

Reviewer: @PixarV а в дататайпе vpc написано, что это дататайп не поддерживается

The following arguments are supported:

* `cidr_block` - (Optional, Forces new resource, String) The IPv4 CIDR block for the VPC.
* `enable_dns_support` - (Optional, Boolean) A flag to enable or disable the DNS support in the VPC.
    * _Default value:_ `true`
* `tags` - (Optional, Map of strings) Key-value pairs to assign to the resource. If a provider [default_tags configuration block][default-tags] is used, tags with matching keys will overwrite those defined at the provider level.

## Attribute Reference

### Supported attributes

In addition to all arguments above, the following attributes are exported:

* `arn` - (String) The Amazon Resource Name (ARN) of the VPC.
* `default_network_acl_id` - (String) The ID of the network ACL created by default on VPC creation.
* `default_security_group_id` - (String) The ID of the security group created by default on VPC creation.
* `default_route_table_id` - (String) The ID of the route table created by default on VPC creation.
* `id` - (String) The ID of the VPC.
* `instance_tenancy` - (String) The allowed tenancy of instances.

> * `main_route_table_id` (String) - The ID of the main route table associated with this VPC. Note that you can change a VPC's main route table by using an [`aws_main_route_table_association`](main_route_table_association.md).

Reviewer: ```suggestion
* `main_route_table_id` - (String) The ID of the main route table associated with this VPC. Note that you can change a VPC's main route table by using an [`aws_main_route_table_association`](main_route_table_association.md).
```

The following arguments are supported:

* `cidr_block` - (Optional, Forces new resource) (String) The IPv4 CIDR block for the VPC.

> * `enable_dns_support` - (Optional) (Bool) A boolean flag to enable/disable DNS support in the VPC.

Reviewer: ```suggestion
* `enable_dns_support` - (Optional, Boolean) Indicates whether to enable/disable DNS support in the VPC.
```
